src/common.py

```diff
@@ -1,51 +1,60 @@
+'''This is mainly used to create contextInfo'''
+
 import logging
 import os
-
 import yaml
 
-logger = logging.getLogger(__name__)
-
 
 class Singleton(type):
+    '''Singleton'''
+
     _instances = {}
 
     def __call__(cls, *args, **kwargs):
         if (cls, args, frozenset(kwargs.items())) not in cls._instances:
-            cls._instances[(cls, args, frozenset(kwargs.items()))] = super(Singleton, cls).__call__(*args, **kwargs)
+            cls._instances[(cls, args, frozenset(kwargs.items()))] \
```

Comment:
I don't actually think this long line change makes the code more readable, despite it passing the long-line test.

Reply:
I agree and it doesn't make it more readable on gitub. I will change it back